### DISCUSSION TITLE
Firmware 50520230320

### DISCUSSION THREAD

**Discussion by couriersud** [roleoroleo/sonoff-hack#143] [General]
For my S-CAM the eWeLink app shows that a new firmware labelled `50520230320`is available. 
Currently `50420230103` is installed.
Does anybody here has already tried the new firmware? 
Any experience to share?

**Comment by felipebraz** [comment]
Also wants to know

**Comment by sonped** [comment]
I have just unpacked my camera and until now it seems to work fine with these versions:
![image](https://github.com/user-attachments/assets/f57e020e-a069-4fac-816a-78d8ac13b351)

Note this is my first encounter with a camera though, so still haven't setup/integrated it.

**Comment by PhRaucq** [comment]
Hello,

I have the same GK-200MP2-B camera and the app annouce an actual version 50520230320.
But where do you get the build? I do not have it in the app.

Anyhow, it works fine with the file GK-200MP2-B_0.1.7

**Comment by sonped** [comment]
See readme file section "getting started" point 7.

You will then find a base version on the webpage.

________________________________
